medium-tests/src/test/java/mediumtest/sca/ScaIssueStatusChangeMediumTests.java

```diff
@@ -0,0 +1,670 @@
+      .withPackageName("com.example.vulnerable")
+      .withPackageVersion("1.0.0")
+      .withTransitions(List.of(
+        ServerScaIssue.Transition.ACCEPT
+      ));
+
+    var server = harness.newFakeSonarQubeServer()
+      .withProject(PROJECT_KEY, project -> project.withBranch(BRANCH_NAME))
+      .start();
+    var backend = harness.newBackend()
+      .withSonarQubeConnection(CONNECTION_ID, server.baseUrl(), storage -> storage
+        .withProject(PROJECT_KEY, project -> project.withMainBranch(BRANCH_NAME, branch -> branch.withScaIssue(scaIssue))))
+      .withBoundConfigScope(CONFIGURATION_SCOPE_ID, CONNECTION_ID, PROJECT_KEY)
+      .start();
+
+    var params = new ChangeScaIssueStatusParams(CONFIGURATION_SCOPE_ID, scaIssueKey, DependencyRiskTransition.ACCEPT, "   ");
+    var scaService = backend.getScaService();
+
+    assertThat(scaService.changeStatus(params))
+      .failsWithin(2, TimeUnit.SECONDS)
+      .withThrowableOfType(ExecutionException.class)
+      .withCauseExactlyInstanceOf(ResponseErrorException.class)
+      .withMessageContaining("Comment is required for ACCEPT, FIXED, and SAFE transitions");
+  }
+
+  @SonarLintTest
+  void it_should_handle_sca_issue_with_different_severities(SonarLintTestHarness harness) {
+    var scaIssueKey = UUID.randomUUID();
+    var scaIssue = aServerScaIssue()
+      .withKey(scaIssueKey)
+      .withType(ServerScaIssue.Type.VULNERABILITY)
+      .withSeverity(ServerScaIssue.Severity.HIGH)
+      .withStatus(ServerScaIssue.Status.OPEN)
+      .withPackageName("com.example.critical")
+      .withPackageVersion("1.0.0")
+      .withTransitions(List.of(
+        ServerScaIssue.Transition.CONFIRM,
+        ServerScaIssue.Transition.ACCEPT
+      ));
+
+    var server = harness.newFakeSonarQubeServer()
+      .withProject(PROJECT_KEY, project -> project.withBranch(BRANCH_NAME))
+      .start();
+    var backend = harness.newBackend()
+      .withSonarQubeConnection(CONNECTION_ID, server.baseUrl(), storage -> storage
+        .withProject(PROJECT_KEY, project -> project.withMainBranch(BRANCH_NAME, branch -> branch.withScaIssue(scaIssue))))
+      .withBoundConfigScope(CONFIGURATION_SCOPE_ID, CONNECTION_ID, PROJECT_KEY)
+      .start();
+
+    var params = new ChangeScaIssueStatusParams(CONFIGURATION_SCOPE_ID, scaIssueKey, DependencyRiskTransition.ACCEPT, "Critical issue accepted");
+    var scaService = backend.getScaService();
+
+    var response = scaService.changeStatus(params);
+    assertThat(response).succeedsWithin(Duration.ofSeconds(2));
+    waitAtMost(2, SECONDS).untilAsserted(() -> {
+      server.getMockServer()
+        .verify(WireMock.postRequestedFor(urlEqualTo("/api/v2/sca/issues-releases/change-status"))
+          .withHeader("Content-Type", equalTo("application/json; charset=UTF-8"))
+          .withRequestBody(equalToJson(
+            "{\"issueReleaseKey\":\"" + scaIssueKey + "\",\"transitionKey\":\"ACCEPT\",\"comment\":\"Critical issue accepted\"}")));
```

Comment:
You could use text blocks in all the test cases, I think it will make it a bit less verbose